features/visitor_can_see_articles_on_index_page.feature

```diff
@@ -5,7 +5,15 @@ Feature: List articles on the landing page
     I would like to be able to see the articles listed on a page
 
     Background: 
-         Given the following articles exist
+        Given the following categories exist
```

Comment:
No need to create categories like this, take a look at the step definition for creating the articles

Reply:
done